Workbooks/TSG/Performance Counter TSG/Performance Counter TSG.workbook

```diff
@@ -0,0 +1,231 @@
+{
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+  "version": "Notebook/1.0",
+  "isLocked": false,
```

Comment:
True?

Reply:
I don't know. Is it? I have copied from the data that saved in shared.

Reply:
Should be true - the report opens up locked.